.circleci/config.yml

```
@@ -35,15 +35,14 @@ jobs:
       - run:
           name: "Start microk8s"
           command: |
-            sudo microk8s.start
-            sudo microk8s.status --wait-ready
-            sudo microk8s.enable dns
-            kubectl get all --all-namespaces
+            (echo "microk8s.start; microk8s.status --wait-ready; microk8s.enable dns rbac ") | sudo -H -s -E  
       - run: 
           name: "Install Helm"
-          command: |
+          command: |            
```

Not sure if these whitespaces can cause issues but it's yaml so maybe:

```suggestion
          command: |
```